templates/webpay/plus_deferred/status.html

```diff
@@ -0,0 +1,26 @@
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>Ejemplos Webpay - Estado de Transacci&oacute;n</title>
+    {% include 'include/header.html' %}
+</head>
+
+<body class="container">
+<nav aria-label="breadcrumb">
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item"><a href="#">Ejemplos Patpass Webpay - Estado de Estado</a></li>
```

Comment:
same here, ups forgot to write in english the comment above this one, but I'm sure you'll forgive me